ledger-tool/src/main.rs

```diff
@@ -1425,10 +1425,16 @@ fn main() {
                     .help("Include sysvars too"),
             )
             .arg(
-                Arg::with_name("exclude_account_data")
-                    .long("exclude-account-data")
+                Arg::with_name("no_accounts_contents")
+                    .long("no-accounts-contents")
                     .takes_value(false)
-                    .help("Exclude account data (useful for large number of accounts)"),
+                    .help("Do not print contents of each account, which is very slow with lots of accounts."),
+            )
```

Comment:
Github's diff makes it look like this is a changed value, but this is the new arg.